addon/mixins/run.js

```diff
@@ -371,25 +373,23 @@ export default Mixin.create({
     }
 
     let tick = () => callback.call(this, next);
+    let token = getNextToken();
 
-    if (label) {
-      assert(`The label provided to \`pollTask\` must be unique. \`${label}\` has already been registered.`, !pollTaskLabels[label]);
-      pollTaskLabels[label] = true;
+    pollTaskTokens[token] = true;
 
-      getOrAllocate(this, '_pollerLabels', Array).push(label);
-    }
+    getOrAllocate(this, '_pollerTokens', Array).push(token);
```

Comment:
do we ever remove the `token` from this `_pollerTokens` array?

Reply:
We never remove the token from the array, but the tokens are used only right before the object's destruction, in `willDestroy`. I suspect we don't need to explicitly remove the token from the `_pollerTokens` array, since it won't be used post destruction.

Reply:
I'm going to look at removing this in my next refactor, which will include moving us over to using weakmaps for managing the task related metadata.